Final_project_2/frontend/css/style.css

```diff
@@ -463,20 +463,15 @@ label  {
   margin-right: 30px;
 } 
 
-
```

Comment:
there's a lot of inconsistent whitespacing in this file; here I've removed just a couple of the superfluous empty lines. I recommend you look into installing a linter that would  automatically format your CSS and HTML files or use a code editor (IDE) that does that for you automatically.

Reply:
hey this is bad its ugly change it back

Reply:
okay done now thank you!